megatron/arguments.py

```diff
@@ -270,55 +347,79 @@ def _add_initialization_args(parser):
                        help='Initial learning rate. Depending on decay style '
                        'and initial warmup, the learing rate at each '
                        'iteration would be different.')
-    group.add_argument('--lr-decay-style', type=str, default='linear',
+    group.add_argument('--lr-decay-style',
+                       type=str,
+                       default='linear',
                        choices=['constant', 'linear', 'cosine', 'exponential'],
                        help='Learning rate decay function.')
-    group.add_argument('--lr-decay-iters', type=int, default=None,
+    group.add_argument('--lr-decay-iters',
+                       type=int,
+                       default=None,
                        help='number of iterations to decay learning rate over,'
                        ' If None defaults to `--train-iters`')
-    group.add_argument('--min-lr', type=float, default=0.0,
+    group.add_argument('--min-lr',
+                       type=float,
+                       default=0.0,
                        help='Minumum value for learning rate. The scheduler'
                        'clip values below this threshold.')
-    group.add_argument('--warmup', type=float, default=0.01,
+    group.add_argument('--warmup',
+                       type=float,
+                       default=0.01,
                        help='Percentage of total iterations to warmup on '
                        '(.01 = 1 percent of all training iters).')
-    group.add_argument('--override-lr-scheduler', action='store_true',
+    group.add_argument('--override-lr-scheduler',
+                       action='store_true',
                        help='Reset the values of the scheduler (learning rate,'
                        'warmup iterations, minimum learning rate, maximum '
                        'number of iterations, and decay style from input '
                        'arguments and ignore values from checkpoints. Note'
                        'that all the above values will be reset.')
-    group.add_argument('--use-checkpoint-lr-scheduler', action='store_true',
-                       help='Use checkpoint to set the values of the scheduler '
-                       '(learning rate, warmup iterations, minimum learning '
-                       'rate, maximum number of iterations, and decay style '
-                       'from checkpoint and ignore input arguments.')
+    group.add_argument(
+        '--use-checkpoint-lr-scheduler',
+        action='store_true',
+        help='Use checkpoint to set the values of the scheduler '
+        '(learning rate, warmup iterations, minimum learning '
+        'rate, maximum number of iterations, and decay style '
+        'from checkpoint and ignore input arguments.')
 
     return parser
 
 
 def _add_checkpointing_args(parser):
     group = parser.add_argument_group(title='checkpointing')
 
-    group.add_argument('--save', type=str, default=None,
+    group.add_argument('--save',
```

Comment:
It looks like auto-formatting happened at some point and it's blowing up the diff size :-(. I wonder if we can quickly revert these by running whatever formatter Megatron uses?